docs/developer-guide.rst

```diff
@@ -1,82 +1,84 @@
+to create c2cgeoform-based applications. This scaffold produce a fully
+functional c2cgeoform-base project: the c2cgeoform_demo project.
+
+When running code checks and tests, these jobs are first run on the c2cgeoform
+framework itself where after the c2cgeoform_demo project is generated in `.build`
+folder and finally checks and tests are launched on this project too.
+
+Note that you should never alter the c2cgeoform_demo project itself but the
+c2cgeoform scaffold and regenerate the c2cgeoform_demo project.
+
+.. _developer-guide Prerequisites:
 
 Prerequisites
 ~~~~~~~~~~~~~
 
 You need to install PostgreSQL and PostGIS. On Ubuntu, the packages
 ``postgresql-server-dev-9.3`` and ``python-dev`` are required.
 
-You need to create a role. For example:
+Clone the project
+~~~~~~~~~~~~~~~~~
 
 .. code-block:: shell
 
-   sudo su - postgres
-   createuser --pwprompt c2cgeoform
+   git clone git@github.com:camptocamp/c2cgeoform.git
+   cd c2cgeoform
 
-You need to create a PostGIS database. For example:
+Run the checks
+~~~~~~~~~~~~~~
 
 .. code-block:: shell
 
-   createdb --owner=c2cgeoform c2cgeoform
-   psql -d c2cgeoform -c "CREATE EXTENSION postgis;"
+   make check
 
-Configuration
+Run the tests
 ~~~~~~~~~~~~~
 
-Edit ``tests.ini`` and modify the SQLAlchemy database connection string
-as appropriate.
-
-For example:
-
-.. code-block:: ini
-
-   sqlalchemy.url = postgresql://c2cgeoform:c2cgeoform@localhost:5432/c2cgeoform
-
-Installation
-~~~~~~~~~~~~
-
-Install a dev egg of ``c2cgeoform`` in a virtual env:
+Create the tests database:
 
 .. code-block:: shell
 
-   make install
-
-(This creates the virtual env for you.)
-
-Just run ``make`` to know about the possible targets.
+   sudo -u postgres psql -c "CREATE USER \"www-data\" WITH PASSWORD 'www-data';"
 
-Initialize the DB
-~~~~~~~~~~~~~~~~~
+   export DATABASE=c2cgeoform_demo_tests
+   sudo -u postgres psql -d postgres -c "CREATE DATABASE $DATABASE OWNER \"www-data\";"
+   sudo -u postgres psql -d $DATABASE -c "CREATE EXTENSION postgis;"
 
-Initialize the database:
+Run the framework and demo tests:
 
 .. code-block:: shell
 
-   make initdb
-
-Run the tests
-~~~~~~~~~~~~~
+   make test
 
-Run the tests:
+Serve the c2cgeoform_demo project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Create the tests database:
+You need to create a PostGIS database. For example:
 
 .. code-block:: shell
 
-   sudo -u postgres psql -c "CREATE USER \"www-data\" WITH PASSWORD 'www-data';"
-
-   export DATABASE=c2cgeoform_demo_tests
+   export DATABASE=c2cgeoform_demo
    sudo -u postgres psql -d postgres -c "CREATE DATABASE $DATABASE OWNER \"www-data\";"
    sudo -u postgres psql -d $DATABASE -c "CREATE EXTENSION postgis;"
-
-Run the framework and demo tests:
+   make initdb
 
 .. code-block:: shell
 
-   make test
+   make serve
+
+You can now open the demo project in your favorite browser:
+http://localhost:6543/
+
+Here is it, you're ready to develop in c2cgeoform. Make changes in c2cgeoform,
```

Comment:
Et voilou ?

Reply:
Here is it => And there you go

Reply:
you are ready to develop, make changes, run checks and tests in c2cgeoform. And finally see the results in ...